Source/AtomicEditor/Editors/JSResourceEditor.cpp

```diff
@@ -177,6 +177,39 @@ bool JSResourceEditor::OnEvent(const TBWidgetEvent &ev)
         if (ev.ref_id == TBIDC("close"))
         {
             RequestClose();
+        } else if (ev.ref_id == TBIDC("undo")) {
+            // Need to physically send the CTRL/CMD+Z to the browser so that
```

Comment:
Crunchy block of code, happy that can be baked down here without invading a bunch of source files 👍